core/sql/regress/seabase/dml.sb028

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
Thanks.  Will include the copyright notice.

On Mon, Sep 21, 2015 at 11:03 AM, DaveBirdsall <notifications@github.com>
wrote:

> In core/sql/regress/seabase/dml.sb028
> <https://github.com/apache/incubator-trafodion/pull/87#discussion_r39988941>
> :
>
> > @@ -0,0 +1,11 @@
> > +
>
> As should this
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-trafodion/pull/87/files#r39988941>.
>



-- 
Regards, --Qifan